guardrails/validator_base.py

```diff
@@ -1,9 +1,14 @@
 import inspect
 from collections import defaultdict
+from enum import Enum
 from typing import Any, Callable, Dict, List, Literal, Optional, Tuple, Type, Union
 
 from pydantic import BaseModel, Field
 
+class OnFailAction(Enum):
```

Comment:
Can you make the enum support all `on_fail`actions listed here -> https://www.guardrailsai.com/docs/concepts/output/#%EF%B8%8F-specifying-corrective-actions